go/vt/vttablet/tabletmanager/vreplication/vplayer.go

```diff
@@ -217,6 +241,9 @@ func (vp *vplayer) applyStmtEvent(ctx context.Context, event *binlogdatapb.VEven
 }
 
 func (vp *vplayer) applyRowEvent(ctx context.Context, rowEvent *binlogdatapb.RowEvent) error {
+	if err := vp.updateFKCheck(ctx, rowEvent.Flags); err != nil {
+		return err
+	}
```

Comment:
This seems heavy, even though we'll often shortcut the work. And it will impact all vreplication usage. When would this change in the middle of a table copy?